src/game/objects/switch.c

```diff
@@ -24,7 +24,7 @@ static int16_t m_Switch1Bounds[12] = {
     +10 * PHD_DEGREE,
 };
 
-static int16_t m_Switch1BoundsControlled[12] = {
+static int16_t m_Switch_BoundsControlled[12] = {
```

Comment:
Should we add back in the `1` at the end to make sure this is clear? I know it only applies to switch1 though. Maybe we can drop the number 1 from all switch1 variables and rename `m_Switch_Bounds2` to `m_Switch_BoundsUW` etc. `UW` is much more clear than `2`.

I was thinking about something similar for Scion numbering too, but I didn't dig into that.

Reply:
Agreed about UW being much more readable. Ignored Scion pieces.